examples/dashboard_embeddable_examples/public/app.tsx

```diff
@@ -8,7 +8,8 @@
 
 import React from 'react';
 import ReactDOM from 'react-dom';
-import { BrowserRouter as Router, Route, RouteComponentProps, withRouter } from 'react-router-dom';
+import { BrowserRouter as Router, RouteComponentProps, withRouter } from 'react-router-dom';
```

Comment:
@lukeelmers @rshen I'm wondering if there's any value in re-exporting these types from Shared UX, rather than having the two packages repeated everywhere.  It would give us the flexibility to change the types, if necessary, and reduce confusion or potential misuse.

If so, we could just run a refer replace, adding 'type' and switching 'react-router-dom' with the new lib...?

Reply:
No strong opinions from me on that, but I'd say you should only do it if you plan to add an eslint rule to catch anyone who tries to import from `react-router-dom` directly. And you'd probably want to have a plan for how to deal with new APIs that may get added in the future (would shared ux just export everything, or only certain things? etc)

Reply:
[6438c9c](https://github.com/elastic/kibana/pull/150357/commits/6438c9cdf06e2dfb9e1f9cc5f0e11ae0bc182779) for eslint rule added thanks!